BSP/sharpMemoryLCD/LS013B7DH03.c

```diff
@@ -241,11 +246,26 @@ bool sharpMemoryLCD_printTextLine(uint8_t line, const char *text, uint8_t nbChar
                     // we need to update pixels inside a byte.
                     // modify the pixel/bit in the correct byte of the buffer
                     // by taking the correct bit in the correct byte of font table
-                    pixelBuf[y*SCREEN_WIDTH + screenX/NB_BIT_PER_BYTE] |= (1<<(screenX % NB_BIT_PER_BYTE) & Font16.table[indexCurrentByteInFont]);
+                    bool fontPixelBit = (1<<((NB_BIT_PER_BYTE-1)-(x % NB_BIT_PER_BYTE))) & Font16.table[indexCurrentByteInFont];
```

Comment:
This deserve intermediate var with clear names.
It is hard to grasp what is happening on these multiples instructions.